draft-ietf-teep-architecture.md

```diff
@@ -1288,8 +1287,7 @@ for protecting the resource usage allocated for Trusted Component management.
 ## CA Compromise or Expiry of CA Certificate
 
 A root CA for TAM certificates might get compromised or its certificate might
-expire, or a Trust Anchor other than a root CA certificate may also expire or
-be compromised.
+expire.
```

Comment:
I think this change is incorrect.  Line 1287 is about expiry of any CA, not just a root CA.   The change limits it to only root CA and contradicts the section title.

Reply:
Restored in my version.